### ISSUE TITLE
Baseline issue

### ISSUE THREAD

**Issue by WangLu95** [hirotomusiker/CLRerNet#11]
I am using the same baseline CLRNet as you, and I am using their original code for training and testing. However, F1 cannot reach 80.47. I would like to ask how you conducted the baseline experiment? Is it through their original code and parameters? Could you please teach me? Thank you very much.

**Comment by hirotomusiker** [comment]
Indeed, we also tried to train and reproduce 80.47 with the official repo and ended up finding F1 is around 80.1 and it varies with different seeds. (That is why we decided to do benchmark with multiple seeds (N=5) and employ the statistical F1 values in our paper.)
We conducted CLRNet training with omitting redundant frames (see Section 4 of [our paper](https://arxiv.org/abs/2305.08366)) and set it as the new baseline (80.86±0.06, N=5).  The F1 score of 80.47 in README.md is the value from the CLRNet paper. 
Thank you.
